chapters/ch25.md

```diff
@@ -1,31 +1,32 @@
 # Techniques for reducing avoidable bias
 ->
-
+# Các kĩ thuật để giảm độ chệch có thể tránh được
 
 If your learning algorithm suffers from high avoidable bias, you might try the following techniques:
 ->
-
+Nếu thuật toán học của bạn gặp vấn đề với độ chệch tránh khỏi lớn, bạn có thể thử những kĩ thuật sau:
 
 * **Increase the model size** ​(such as number of neurons/layers): This technique reduces bias, since it should allow you to fit the training set better. If you find that this increases variance, then use regularization, which will usually eliminate the increase in variance.
 ->
+* **Tăng kích thước mô hình** (ví dụ như số lượng neuron/tầng): Kĩ thuật này làm giảm độ chệch, vì nó cho phép fit tập huấn luyện tốt hơn. Nếu bạn thấy việc này làm tăng phương sai, hãy sử dụng regularization, vốn thường loại bỏ việc tăng phương sai.
 
 
 * **Modify input features based on insights from error analysis​**: Say your error analysis inspires you to create additional features that help the algorithm eliminate a particular category of errors. (We discuss this further in the next chapter.) These new features could help with both bias and variance. In theory, adding more features could increase the variance; but if you find this to be the case, then use regularization, which will usually eliminate the increase in variance.
 ->
-
+* **Thay đổi các đặc trưng đầu vào dựa trên nhận thức rõ có được từ phân tích lỗi ​**: Giả sử việc phân tích lỗi gợi ý bạn rằng hãy tạo thêm những đặc trưng bổ sung nhằm giúp thuật toán có thể loại bỏ một kiểu lỗi đặc thù. (Chúng ta sẽ bàn vấn đề này kĩ hơn ở chương sau.) Những đặc trưng mới này có thể hiệu quả với cả độ chệch và phương sai. Trong lý thuyết, thêm đặc trưng có thể làm tăng phương sai; nhưng nếu bạn thấy đúng là điều đó xảy ra, hãy sử dụng regularization, vốn thường loại bỏ việc tăng phương sai.
 
 * **Reduce or eliminate regularization​** (L2 regularization, L1 regularization, dropout): This will reduce avoidable bias, but increase variance.
 ->
-
+* **Giảm hoặc loại bỏ regularization​** (L2 regularization, L1 regularization, dropout): Việc này sẽ làm giảm độ chệch có thể tránh được, nhưng sẽ làm tăng phương sai.
 
 * **Modify model architecture​** (such as neural network architecture) so that it is more suitable for your problem: This technique can affect both bias and variance.
 ->
-
+* **Thay đổi kiến trúc mô hình** (ví dụ như kiến trúc mạng neural) để nó trở nên phù hợp hơn với bài toán của bạn: Kĩ thuật này có thể ảnh hưởng đến cả độ chệch và phương sai.
 
 One method that is not helpful:
 ->
-
+Một phương pháp không hiệu quả:
 
 * **Add more training data​**: This technique helps with variance problems, but it usually has no significant effect on bias.
 ->
-
+* **Thêm dữ liệu huấn luyện​**: Kĩ thuật này có ích với vấn đề  phương sai, nhưng nó thường không có tác dụng gì đáng kể với độ chệch.
```

Comment:
hai dấu cách "đề  phương"

Reply:
> @ngcthuong chữ "chuẩn hóa" ở đây không phải là "tiêu chuẩn hóa" như standardization. Khi mình nói chuẩn hóa ý niệm của mình nó có nét tương tự như "normalization" vậy.
> 
> Có điểm khác biệt là normalization sẽ scale lại data input đầu vào, VD chuyển hết range của các features về [0, 1], còn regularization là một kĩ thuật để tránh overfitting, thay vì scaling lại data thì nó điều chỉnh hàm predict. Thực ra theo một nghĩa nào đó, việc này cũng là "chặn trên chặn dưới", áp đặt một cái range cho các giá trị của hệ số của hàm predict, tức là nó cũng "chuẩn hóa" hệ số vào trong một dải.
> 
> (Đây chỉ là trao đổi chút về mặt chữ/ngữ nghĩa, còn đã thống nhất trong glossary thì sẽ để nguyên. Dùng chung là chuẩn hóa thì sẽ lẫn lộn, mà chưa thể có được term nào thay thế thì để nguyên thôi).

Để mình giải thích thêm một chút. Thuật ngữ regularization dùng trong optimization từ trước deep learning hay ML. Và nghĩa khái quát rộng hơn - như mình nói- là giới hạn không gian nghiệm theo một điều kiện ràng buộc nào đó. Khác biệt nhiều với nghĩa chuẩn hoá hay là "normalizartion"

Vì trong các thuật ngữ khác như "total variation regularization" mình dịch regularization thảnh "chuẩn hoá" thì bị lệch với nghĩa gốc. Nên mình nghĩ không nên dùng tử "chuẩn hoá" là lý do như vậy.

Reply:
Và từ revgularization hay được dùng theo nghĩa tương ứng với "constraint"

Reply:
@ngcthuong nó có "tác dụng" giống "constraint" còn nghĩa thì không hẳn giống.
"Chuẩn hóa" thì chắc chắc không đúng rồi.

Tạm thời bỏ qua các dịch "regularization" ở chương này, giữ nguyên là "regularization" như từ đầu. Mình có thể tạo hẳn một PR khác để bàn riêng về từ này, khi chấp nhận thì tạo một commit để thay toàn bộ các từ từ đầu.